### ISSUE TITLE
Icons in event list are missing alt text

### ISSUE THREAD

**Issue by carrythebanner** [Shift2Bikes/shiftcal#144]
Steps:
1. Open calendar: http://www.shift2bikes.org/fun/
2. Press any ride to expand and show its details
3. Inspect the icons for location, start time, ride leader, etc. 

Result:
Icons do not have alt text.

**Comment by carrythebanner** [comment]
Merged and deployed.
